### ISSUE TITLE
Import Extent (Physical Description)

### ISSUE THREAD

**Issue by mark-dce** [curationexperts/alexandria-v2#101]
Path in MODS file:
mods:physicalDescription/extent 

Predicate:
RDF::DC.extent

Ruby name:
extent

data type:
string

**Comment by val99erie** [comment]
We should discuss this before anyone works on this.  It is duplcate of work done in #49 and #88.  I'll mark it as blocked until we discuss.

**Comment by val99erie** [comment]
The specific question I want to discuss is whether this property should map to RDF::DC.extent or RDF::Vocab::MODS.physicalExtent.  We have already done it both ways.

**Comment by jcoyne** [comment]
DC.extent is my preference because MODSRDF.physicalExtent has a domain of ModsResource (see:  http://www.loc.gov/standards/mods/modsrdf/v1/#physicalExtent). We aren't meeting that constraint

**Comment by val99erie** [comment]
ok, let's do DC.extent.  I'll unblock the ticket
